backend/main/test/Group/Domain/Entities/Food.test.ts

```diff
@@ -36,99 +37,81 @@ describe("Food Object", () => {
 			});
```

Comment:
Technically, you don't need to use spread syntax.
This is FAI, and it's up to your preference.

```typescript
const food = Food.create(foodId, requiredFoodProps);
```

Reply:
Thank you! I change it!